docs/onboarding/transactions.md

```diff
@@ -49,6 +50,13 @@ E.g. To get the public key of a different account on your node
 ironfish wallet:address Savings
 ```
 
+### View transactions
```

Comment:
Super nit, but unless "View transactions" is supposed to be a subsection of "Receive a transaction", I believe it should be an h2.

Reply:
oh yeah, good catch

Reply:
Fixed